bedrock/settings/base.py

```diff
@@ -110,9 +111,11 @@ def path(*args):
 
 # Tells the product_details module where to find our local JSON files.
 # This ultimately controls how LANGUAGES are constructed.
-PROD_DETAILS_DIR = path('lib', 'product_details_json')
 PROD_DETAILS_CACHE_NAME = 'product-details'
 PROD_DETAILS_CACHE_TIMEOUT = 60 * 15  # 15 min
+default_pdstorage = 'PDDatabaseStorage' if PROD else 'PDFileStorage'
```

Comment:
Just no reason to create all of that confusion with all of the contributors with existing copies. They don't often run `./manage.py migrate`. This way their day is uninterrupted.